packages/database/src/iriToHash.test.ts

```diff
@@ -0,0 +1,15 @@
+import {iriToHash} from './iriToHash';
+
+describe('iriToHash', () => {
+  it('returns a hash', () => {
+    const hash = iriToHash('https://example.com');
+    expect(hash).toBe('c984d06aafbecf6bc55569f964148ea3');
+  });
+
+  it('returns a hash of the same length', () => {
```

Comment:
You could add the `expect()` to the first test (because 'returns a hash' implies that it must be a valid hash, i.e. also be of a specific length)?